reps/config.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  rootPath: "../../mozilla-central",
+  devtoolsPath: "../../mozilla-central/devtools",
```

Comment:
Hi @janodvarko,

But people might already have their own directory structure, like @steveck-chung and I. So I think configuring the paths might be easier. How do you think?